module_utils/network_lsr/argument_validator.py

```diff
@@ -2568,65 +2568,55 @@ def _ipv6_is_not_configured(connection):
 
 class IPRouteUtils(object):
 
+    # iproute2 does not care much about the valid characters of a
+    # table alias (it doesn't even require UTF-8 encoding, the only
+    # forbidden parts are whitespace).
+    #
+    # We don't allow such flexibility. Aliases must only contain a
+    # certain set of ASCII characters.
```

Comment:
Please add a justification for diverting from iproute2 here.

Reply:
done.